### PR TITLE
Better init

### PR DESCRIPTION
**Pull request by technoweenie**
* [x] Basic `git lfs init` tests in #446
* [x] confirm `(MISSING)` is not in the `git lfs init` output.
* [x] Add `git lfs init --force` to force writing the git lfs config.
* [x] Document `git lfs init --force`
* [x] Add `git lfs uninit`
* [x] Document `git lfs uninit`

### CODE REVIEW

**Comment by technoweenie** [comment]
Ready for :eyes:

File: commands/commands_uninit.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+var (
+	uninitCmd = &cobra.Command{
+		Use:   "uninit",
+		Short: "Clear the Git LFS configuration",
+		Run:   uninitCommand,
+	}
+
+	uninitHooksCmd = &cobra.Command{
+		Use:   "hooks",
+		Short: "Clear the only Git hooks for the current repository",
```
Comment (rubyist) [inline commands/commands_uninit.go]: Should this say "Clear only the ..."?

Comment (technoweenie) [inline commands/commands_uninit.go]: Er yup!

File: docs/man/git-lfs-uninit.1.ronn [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+## SYNOPSIS
+
+`git lfs uninit`
+
+## DESCRIPTION
+
+Perform the following actions to remove the Git LFS configuration:
+
+* Remove the "lfs" clean and smudge filters from the global Git config.
+* Uninstall a Git LFS pre-push hook if run from inside a Git repository.
```
Comment (rubyist) [inline docs/man/git-lfs-uninit.1.ronn]: Maybe "Uninstall the ..."

**Comment by rubyist** [comment]
:+1:

**Pull request merged**